packages/strapi-design-system/src/DatePicker/DatePickerCalendar.js

```diff
@@ -71,7 +71,7 @@ export const DatePickerCalendar = ({
           </Box>
           <RawTable colCount={7} rowCount={weeks.length + 1} initialCol={activeCol} initialRow={activeRow} role="grid">
             <RawThead>
-              <RawTr>
+              <RawTr aria-rowindex={1}>
```

Comment:
Not sure to understand why you've added this static rowindex?

Reply:
because we had some warnings inside the tests, aria-rowindex is a prop required